src/components/ui/Breadcrumb/breadcrumb.scss

```diff
@@ -41,21 +66,94 @@
 
     [data-breadcrumb-divider] {
       height: var(--fs-spacing-3);
-      margin: 0 var(--fs-spacing-2) 0 var(--fs-spacing-1);
+      margin: 0 var(--fs-spacing-1) 0 var(--fs-spacing-1);
       border-left: var(--fs-border-width-default) solid var(--fs-border-color-light);
     }
 
     [data-store-link] {
-      display: inline;
       padding: 0;
 
       &:visited {
         color: var(--fs-color-link);
       }
     }
 
+    [data-store-dropdown-button] {
+      display: inline;
+      margin: calc(var(--fs-spacing-0) / 4);
+      padding: 0;
+      padding: 0 calc(var(--fs-spacing-0) / 2);
+      color: var(--fs-color-link);
+      letter-spacing: var(--fs-spacing-0);
+      background-color: #0000;
```

Comment:
We don't seem to have a color token for this black. I wonder if we can use the darkest one we have (`--fs-color-neutral-7`)... 
🤔

Reply:
It's not black, it's transparent, I couldn't find a token to transparent colour

Reply:
Oh, I see, the fourth 0 is the transparency. Then, what do you think of using the `transparent` value? I think it's clearer. I was confused when I saw there was a "black" (not really) background somewhere and I wasn't seeing it live.

Reply:
You can just replace for `transparent` :)